contrib/Joshua/scripts/bindingTest.sh

```diff
@@ -3,6 +3,8 @@ SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 ulimit -S -c unlimited
 
 unset FDB_NETWORK_OPTION_EXTERNAL_CLIENT_DIRECTORY
+unset FDB_CLUSTER_FILE
```

Comment:
Why are you unsetting this variable? It is being passed and set within the calling script and needed by us. How are you trying to set the location of the Cluster file?

Reply:
Ooooh, good catch!!  This was one of our early trial while debugging this issue.  I do not think we need this change because all binding tests explicitly use the `fdb.cluster` file dynamically generate.  Let me rerun our test without this change and update the PR if we don't need this.

Reply:
I verified that this change was not necessary.  I reverted the change.